src/Project/Sugcon2024/Sugcon/src/components/List Components/SpeakersGrid.tsx

```diff
@@ -128,23 +127,26 @@ export const Default = (props: PeopleGridProps): JSX.Element => {
   }
 
   return (
-    <Box w="100%" mt={20} className={styles}>
-      <Box w="80%" pt={10} m="auto">
-        {(isEditorActive() || props.fields?.Headline?.value !== '') && (
```

Comment:
This check was there to make sure the headline field shows in editor mode, but hides the heading field if the field is empty in regular mode.  Otherwise you still have the space from an empty h2 tag.

Reply:
@ericsanner good call... I made a change to the whole <Heading /> usage so that is uses the `JssText` component, which handles the conditional rendering.  If the field is empty and it's not pageEditing mode, the tag is not rendered.  If the field is empty and it's in pageEditing mode, the tag is rendered so it can be edited.

```js
<Heading as={JssText} field={props.fields.Headline} tag="h2" size="lg" />
```

That way the logic of showing and hiding the tag is offloaded to the JssText component and the rendering is handled by Chakra Heading component.  JssText allows for a `field` and `tag` attribute as well, which are implemented here.